app/src/main/java/io/neurolab/main/output/visual/SpaceAnimationVisuals.java

```diff
@@ -35,14 +42,46 @@ public static void spaceAnim(View view) {
         handleIndicator(view);
     }
 
+    public static void stopAnim() {
+        try {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
Let's upgrade project minSDK to 23. Lower version has many problems with OTG. What's the version of your device?

Reply:
Android 8.0.0

Reply:
Then no problem for you to work with right? Let's upgrade minSDK version here.

Reply:
Yup. Will do.